src/tests/multihost/docs/layout.rst

```diff
@@ -1,13 +1,56 @@
-layout
+Layout
 ======
-This doc provides the layout of SSSD pytest framework and test suites directory.
+* This doc provides layout of sssd pytest framework and test suites directory.
+
 
 module
-------
-* sssd.testlib
+-------
 
-  This is the main top directory under which there are subdirectories containing various shared functions required to write tests using pytest
+* sssd.testlib
+        * This is the main top directory under which there are subdirectories
+          containing various shared functions required to write tests using
+          pytest.
 
 * sssd.testlib/common
+        * This directory contains modules related to configuring different
+          services like 389-ds, kerberos, samba, Joining to AD, SSH Login,
+          creating posix uses on 389-ds, Creating kerberos users in kerberos
+          database.
+
+* sssd.testlib/common/ipa
+        * This directory contains shared functions related to ipa like setting
+          up chrony, adding hbac rules, mapping external groups to posix
+          groups.
+
+
+Test Directories
+----------------
+
+* sssd/src/tests/multihost
```

Comment:
It might be better to reference the path sssd/src/tests/multihost as a standalone item
and drop "sssd/src/tests/multihost/" from the following items to describe just as: ad, admultidomain, adsites ...
The doc will be more useful for downstream.

Reply:
can we do it in ```.rst``` format?
if you have an example can you link it here?

Reply:
It is just a text right?

Reply:
* sssd/src/tests/multihost
        * This is the \<parent\> directory containing all the pytest test suites.